app/src/app/[lng]/[inventory]/CapTab/ActionDrawer.tsx

```diff
@@ -41,7 +42,7 @@ export function ActionDrawer({
   onClose: () => void;
   t: TFunction;
 }) {
-  console.log(action)
+  logger.info(action)
```

Comment:
Should be removed since it's at the top level of the component so it will print on every render.